_maps/shuttles/whiteship_box.dmm

```diff
@@ -11,17 +11,28 @@
 "ad" = (
 /obj/machinery/door/airlock/titanium,
 /obj/docking_port/mobile{
+	callTime = 250;
 	dheight = 0;
 	dir = 2;
 	dwidth = 11;
 	height = 22;
 	id = "whiteship";
 	launch_status = 0;
+	movement_force = list("KNOCKDOWN" = 0, "THROW" = 0);
 	name = "NT Medical Ship";
 	port_direction = 8;
 	preferred_direction = 4;
 	roundstart_move = "whiteship_away";
-	timid = 1;
+	timid = null;
+	width = 35
+	},
+/obj/docking_port/stationary{
```

Comment:
iirc doesn't having two docking_port modules on a template shuttle prevent it from loading?